README.md

```diff
@@ -1,7 +1,8 @@
-# Ice Window Manager (IceWM)
+# Ice Window Manager (IceWM) Research for fix bug : uncorrectly size screen when we using multimonitor and xrandr
```

Comment:
I think that he switch between internal and external monitor when you are in extend mode. 
Screen on internal monitor will become screen on external monitor and otherwise.